File: 프로그래머스/2/12914. 멀리 뛰기/멀리 뛰기.js
function solution(n) {
    var answer = 0;
    
    let arr = [0,1,2];
    
    for(let i=3; i<=n; i++) {
        arr.push((arr[i-1] + arr[i-2])%1234567);
    }
    
    answer = arr[n];
    
    return answer;
}

File: 프로그래머스/2/12914. 멀리 뛰기/멀리 뛰기.cpp
#include <string>
#include <vector>

using namespace std;

long long solution(int n) {
    long long answer = 0;
    
    vector<int> arr;
    
    arr.push_back(0);
    arr.push_back(1);
    arr.push_back(2);
    
    for(int i=3; i<=n ;i++) 
    {
        arr.push_back((arr[i-1] + arr[i-2]) % 1234567);
    }
    
    answer = arr[n];
    
    return answer;
}